docs/website/content/docs/bootstrapping.md

```diff
@@ -254,7 +252,7 @@ Now that we have the manifest, we can simply apply it:
 kubectl apply -f management-plane.yaml
 ```
 
-> NOTE: The templated manifest above is meant to act as a starting point. If customizations are needed to ensure proper setup of your Talos cluster, they should be added before applying.
```

Comment:
nit: The way we have written notes in our other sites is with the blockquote (`>`). We can merge your change, but one thing I have wanted to do for a long time now is create components that render notes in a "standard" way.

We should look at https://github.com/gridsome/gridsome/issues/677, and https://www.npmjs.com/package/@gridsome/vue-remark. Looks like we should be able to make components similar to: https://matcornic.github.io/hugo-learn-doc/cont/shortcodes/. 

e.g.

```markdown
<note>Some note.</note>
```